### PR TITLE
Feature/leave import

### PR DESCRIPTION
**Pull request by cmcfadden**
Added a command to import the sabbatical eligibility list. Also fix an issue rendering the roles pages.

### CODE REVIEW

File: app/Library/UserService.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ public function findOrCreateByEmplid(string $emplid): ?User {
             return $user;
         }
 
-        $user = LDAP::lookupUser($emplid, 'umnemplid');
+        $user = LDAP::lookupUser(str_pad($emplid, 7, 0, STR_PAD_LEFT), 'umnemplid');
```
Comment (jxjj) [inline app/Library/UserService.php:20]: Could you explain a little why this is needed? Does it handle some special case?

File: app/Console/Commands/importSabbaticalEligibilty.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Library\UserService;
+use Illuminate\Console\Command;
+
+class importSabbaticalEligibilty extends Command
```
Comment (jxjj) [inline app/Console/Commands/importSabbaticalEligibilty.php:8]: Nit: I'm used to classes and their files being capitalized. I don't know if it impacts any Laravel functionality.
```suggestion
class ImportSabbaticalEligibilty extends Command
```

File: resources/js/components/Members.vue [hunk 3]
```diff
@@ -371,7 +371,7 @@ export default {
       );
     },
     compositeList: function() {
-      if(this.group.include_child_groups && this.group.child_groups) {
+      if(this.group && this.group.include_child_groups && this.group.child_groups) {
```
Comment (jxjj) [inline resources/js/components/Members.vue:374]: Good catch!

No change request, but wanted to mention that Vite (or maybe it's TS?) supports optional chaining syntax, which is nice for cases like this:
```ts
      if (this.group?.include_child_groups && this.group.child_groups) {
```

**Review by jxjj — APPROVED** [review]
Nice! Thanks for working on this.

**Review by jxjj — APPROVED** [review]
Nice! Thanks for working on this.

File: app/Library/UserService.php (hunk 1 above)
Comment (cmcfadden) [reply app/Library/UserService.php:20]: some emplids start with leading zeros, and LDAP treats them as strings.

File: resources/js/components/Members.vue (hunk 3 above)
Comment (cmcfadden) [reply resources/js/components/Members.vue:374]: yeah, for whatever reason that syntax doesn't read well in my head so I don't usually do it. Can try to make the switch for consistency though.

File: app/Library/UserService.php (hunk 1 above)
Comment (jxjj) [reply app/Library/UserService.php:20]: Makes sense, thanks!

**Pull request merged**